### DISCUSSION TITLE
Can not import giscus into github page

### DISCUSSION THREAD

**Discussion by oucommunity** [giscus/giscus#558] [Q&A] [answered]
i have error below, can you help me ?
![image](https://user-images.githubusercontent.com/106130487/171262023-a1a18518-8354-4e1b-8fc8-961fe1cdf166.png)

**Comment by laymonage** [comment, marked as answer]
Your [giscus.json](https://github.com/oucommunity/sinh-vien-dai-hoc-mo-tphcm/blob/main/giscus.json) is incorrect. You just need to specify the protocol and domain, e.g. `https://oucommunity.github.io`.


See https://github.com/giscus/giscus/issues/380#issuecomment-1026328308.

**Reply by oucommunity** [reply]
It's work, thank you so much. Have a nice day ^^
